Sources/NostrSDK/EventCreating.swift

```diff
@@ -302,7 +302,7 @@ public extension EventCreating {
     ///   - publishedAt: The date of the first time the article was published.
     ///   - keypair: The ``Keypair`` to sign with.
     /// - Returns: The signed ``LongformContentEvent``.
-    func longformContentEvent(withIdentifier identifier: String,
+    func longformContentEvent(withIdentifier identifier: String = UUID().uuidString,
```

Comment:
Unrelated (but similar to what I'm doing with calendars). Making it slightly easier for clients to integrate by not requiring them to provide a unique identifier -- letting the SDK generate it instead.